arangod/Cluster/AgencyCache.cpp

```diff
@@ -0,0 +1,53 @@
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
Do we want to have this pseudo code here in this PR? Would be OK for me, but is it intentional or an oversight?

Reply:
oversight